campaignion_tracking/package/src/ga4.js

```diff
@@ -0,0 +1,426 @@
+/**
+ * Machine readable name.
+ *
+ * Use for referencing tracker specifics like storage keys, channel names, ...
+ */
+export const name = 'ga4'
+
+/**
+ * Mapping of codes to eventNames.
+ *
+ * This also is a whitelist of which events should *also* be recognized as
```

Comment:
This could easily say “list“ instead of “whitelist“ and also I think there is one more also than needed.

Reply:
Have removed it and replaced all occurrences of "whitelist".

Reply:
Thank you!